app/assets/stylesheets/layouts/dark.scss

```diff
@@ -1,81 +1,183 @@
+
+.flash {
+  a, a:visited { color: #DDD; }
+  a:hover { color: #BBB; }
+}
+.flash.breadcrumbs { background-color: $bg_color_flash_breadcrumbs; color: $font_color_flash_breadcrumbs; }
+.flash.success { background-color: $bg_color_flash_success; }
+.flash.error { background-color: $bg_color_flash_error; }
+.flash.inbox { background-color: $bg_color_flash_inbox; color: $font_color_flash_inbox; }
+
+/* - main page */
+$bg_color_page: #212121; // overall page
+$font_color_page: #A2A2A2;
+
+//$font_color_content: $colors_dark; // body
+
+html, body, #content { background-color: $bg_color_page; color: $font_color_page; } //TODO: 'html' and '#content' flags here are unnessecary
+
+$bg_color_spacer_alt: #333333;
+//$font_color_search_box: #FEFEFE;
+$bg_color_search_box: #333333;
+
+.search-box { background-color: $bg_color_search_box; } // TODO: check text on search box is readable
+.spacer-alt { background-color: $bg_color_spacer_alt; }
+
+/* - misc */
+$bg_color_descriptions: #565656; // description subheadings on templates page ("single-description"), *not* posts
+.single-description { background-color: $bg_color_descriptions; }
+
+/* - content headers */
+$bg_color_head: #424242; // primary content header
+$bg_color_sub: #383838; // secondary content header
+$bg_color_subber: #464646; // tertiary content header
+$font_color_head: #C2C2C2;
+$font_color_sub: #C9C9C9;
+$font_color_subber: #DDDDDD;
+
+th, .content-header { background-color: $bg_color_head; color: $font_color_head; }
+.sub { background-color: $bg_color_sub; color: $font_color_sub; }
+.subber, .form-table-ender { background-color: $bg_color_subber; color: $font_color_subber; }
+
+/* - content */
+$bg_color_even: #3E3E3E;
+$bg_color_odd: #474747;
 
-$EVEN_COLOR: #3E3E3E;
-$ODD_COLOR: #474747;
 .even, .post-post, .post-reply:nth-child(even), #post-editor {
-  background-color: $EVEN_COLOR;
+  background-color: $bg_color_even;
 
   blockquote {
-    background-color: calculate-transparent-color($EVEN_COLOR, $ODD_COLOR, 0.2);
+    background-color: calculate-transparent-color($bg_color_even, $bg_color_odd, 0.2);
   }
 }
 .odd, .post-reply:nth-child(odd), .post-post + #post-editor, .post-post + a + #post-editor {
-  background-color: $ODD_COLOR;
+  background-color: $bg_color_odd;
 
   blockquote {
-    background-color: calculate-transparent-color($ODD_COLOR, $EVEN_COLOR, 0.2);
+    background-color: calculate-transparent-color($bg_color_odd, $bg_color_even, 0.2);
   }
 }
 
-#reply-icon-selector, .replace-icon, .icons-box, .icon-credit { background-color: #444D3F; }
-.subber, .flash.breadcrumbs, .form-table-ender {
-  background-color: #464646;
+
+$bg_color_post_description: #5D5D5D;
+//$font_color_post_description: #F3F3F3;
+$bg_color_navheader: #5F5D57;
+//$bg_color_navbutton: $colors_greybrown;
+
+.post-subheader { background-color: $bg_color_post_description; }
+.post-navheader { background-color: $bg_color_navheader; }
+
+// TODO: this doesn't seem to style #post-menu-box, is that ok?
```

Comment:
leftover todo

Reply:
it is probably okay, since I think it's darkish in default. maybe check. it's the menu on a post, under [≡], I think.